src/styles/_catalog.scss

```diff
@@ -14,9 +14,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: all .3s;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties.